jib-core/src/main/java/com/google/cloud/tools/jib/builder/BuildConfiguration.java

```diff
@@ -121,7 +122,7 @@ public Builder setEnvironment(@Nullable Map<String, String> environmentMap) {
       return this;
     }
 
-    public Builder setExposedPorts(@Nullable List<String> exposedPorts) {
+    public Builder setExposedPorts(@Nullable List<PortsWithProtocol> exposedPorts) {
```

Comment:
I took another look at this and I was thinking it might make more sense to just have this either be a `List<PortWithProtocol>` or a `PortsWithProtocol` (no list), where the `PortsWithProtocol` itself can be constructed with a builder that adds single ports and port ranges. This way, we keep the core configuration simple.

Reply:
Are you suggesting just not keeping track of the ranges again and expanding to individual ports?